guides/keras_cv/fine_tune_via_textual_inversion.py

```diff
@@ -0,0 +1,691 @@
+    ],
+    prompts=[
+        "a photo of a {}",
+        "a rendering of a {}",
+        "a cropped photo of the {}",
+        "the photo of a {}",
+        "a photo of a clean {}",
+        "a photo of my {}",
+        "a photo of the cool {}",
+        "a close-up photo of a {}",
+        "a bright photo of the {}",
+        "a cropped photo of a {}",
+        "a photo of the {}",
+        "a good photo of the {}",
+        "a photo of one {}",
+        "a close-up photo of the {}",
+        "a rendition of the {}",
+        "a photo of the clean {}",
+        "a rendition of a {}",
+        "a photo of a nice {}",
+        "a good photo of a {}",
+        "a photo of the nice {}",
+        "a photo of the small {}",
+        "a photo of the weird {}",
+        "a photo of the large {}",
+        "a photo of a cool {}",
+        "a photo of a small {}",
+    ],
+)
+
+"""
+Next we assemble a dataset of groups of our GitHub avatars:
+"""
+
+group_ds = assemble_dataset(
+    urls=[
+        "https://i.imgur.com/yVmZ2Qa.jpg",
+        "https://i.imgur.com/JbyFbZJ.jpg",
+        "https://i.imgur.com/CCubd3q.jpg",
+    ],
+    prompts=[
+        "a photo of a group of {}",
+        "a rendering of a group of {}",
+        "a cropped photo of the group of {}",
+        "the photo of a group of {}",
+        "a photo of a clean group of {}",
+        "a photo of my group of {}",
+        "a photo of a cool group of {}",
+        "a close-up photo of a group of {}",
+        "a bright photo of the group of {}",
+        "a cropped photo of a group of {}",
+        "a photo of the group of {}",
+        "a good photo of the group of {}",
+        "a photo of one group of {}",
+        "a close-up photo of the group of {}",
+        "a rendition of the group of {}",
+        "a photo of the clean group of {}",
+        "a rendition of a group of {}",
+        "a photo of a nice group of {}",
+        "a good photo of a group of {}",
+        "a photo of the nice group of {}",
+        "a photo of the small group of {}",
+        "a photo of the weird group of {}",
+        "a photo of the large group of {}",
+        "a photo of a cool group of {}",
+        "a photo of a small group of {}",
+    ],
+)
+
+"""
+Finally we concatenate the two datasets:
+"""
+train_ds = single_ds.concatenate(group_ds)
+train_ds = train_ds.batch(1).shuffle(
+    train_ds.cardinality(), reshuffle_each_iteration=True
+)
+
+"""
+## Adding a new token to the text encoder
+Next we will create a new text encoder for the StableDiffusion model and add our new
+embedding for '<my-funny-cat-token>' into the model.
+"""
+tokenized_initializer = stable_diffusion.tokenizer.encode("cat")[1]
+new_weights = stable_diffusion.text_encoder.layers[2].token_embedding(
+    tf.constant(tokenized_initializer)
+)
+
+# Get len of .vocab instead of tokenizer
+new_vocab_size = len(stable_diffusion.tokenizer.vocab)
+
+# SAme hack here to get embedding layer from layers[2]
```

Comment:
Maybe update this comment to something like:

# The embedding layer is the 2nd layer in the text encoder